metricbeat/module/vsphere/host/data_test.go

```diff
@@ -0,0 +1,38 @@
+package host
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/libbeat/common"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEventMapping(t *testing.T) {
+
+	event := eventMapping(HostSystemTest, "test")
+
+	assert.EqualValues(t, "test", event["datacenter"])
+
+	cpu := event["cpu"].(common.MapStr)
+
+	cpuUsed := cpu["used"].(common.MapStr)
+	assert.EqualValues(t, 67, cpuUsed["mhz"])
```

Comment:
FYI It should be possible to write this as

```
value, _ := event.Get("cpu.used.mhz")
assert.EqualValues(t, 67, value)
```

Personally, I find using the dotted field notational easier to work with.